apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregne/Beregne.tsx

```diff
@@ -1,19 +1,46 @@
 import { Content, ContentHeader } from '~shared/styled'
 import { TypeStatusWrap } from '../soeknadsoversikt/styled'
-import { Sammendrag } from './sammendrag'
-import styled from 'styled-components'
-import { BehandlingHandlingKnapper } from '../handlinger/BehandlingHandlingKnapper'
-import { NesteOgTilbake } from '../handlinger/NesteOgTilbake'
 import { hentBehandlesFraStatus } from '../felles/utils'
 import { formaterStringDato } from '~utils/formattering'
 import { formaterVedtaksResultat, useVedtaksResultat } from '../useVedtaksResultat'
-import { useAppSelector } from '~store/Store'
-import { Button } from '@navikt/ds-react'
+import { useAppDispatch, useAppSelector } from '~store/Store'
 import { useBehandlingRoutes } from '../BehandlingRoutes'
+import { useEffect, useState } from 'react'
+import { hentBeregning } from '~shared/api/beregning'
+import { oppdaterBeregning, resetBeregning } from '~store/reducers/BehandlingReducer'
+import Spinner from '~shared/Spinner'
+import { RequestStatus } from '~components/behandling/vilkaarsvurdering/utils'
+import { Sammendrag } from '~components/behandling/beregne/Sammendrag'
+import { BehandlingHandlingKnapper } from '~components/behandling/handlinger/BehandlingHandlingKnapper'
+import { Button } from '@navikt/ds-react'
+import { NesteOgTilbake } from '~components/behandling/handlinger/NesteOgTilbake'
+import styled from 'styled-components'
 
 export const Beregne = () => {
   const { next } = useBehandlingRoutes()
   const behandling = useAppSelector((state) => state.behandlingReducer.behandling)
+  const dispatch = useAppDispatch()
+  const [status, setStatus] = useState<RequestStatus>(RequestStatus.notStarted)
```

Comment:
Bør bruke `useApiCall` her også kan vi bruke utils metodene på den som `isPending(val)` i stedet for å sette staten selv.

Reply:
Kommer egen pr